core/blockchain.go

```diff
@@ -1192,8 +1193,15 @@ func (bc *BlockChain) writeKnownBlock(block *types.Block) error {
 	return nil
 }
 
+// EvmTxTraces groups trace data from each executation of tx and left
+// some field to be finished from blockResult
+type EvmTxTraces struct {
```

Comment:
Moving `EvmTxTraces` to `core/types/l2trace.go` might be more concentrated.

Reply:
done